lib/LedgerSMB/Database/Config.pm

```diff
@@ -112,7 +112,7 @@ List of available files defining a Standard of Industry Codes
 
 sub charts_of_accounts {
     ###TODO: Define a parameter to the SQL directory!!
```

Comment:
Perhaps this todo should be dealt with in this change as well?

Reply:
It needs to be done and soon, but I think that as this change moves away from the SQL directory, this isn't for this change.